docs/sources/whatsnew/whats-new-in-v7-3.md

```diff
@@ -149,4 +149,13 @@ See [upgrade notes]({{< relref "../installation/upgrading.md" >}}).
 
 Check out [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md) for a complete list of new features, changes, and bug fixes.
 
+## Patch release notes
 
+- [Grafana 7.3.0 release notes]({{< relref "../release-notes/release-notes-7-3-0.md" >}})
+- [Grafana 7.3.1 release notes]({{< relref "../release-notes/release-notes-7-3-1.md" >}})
+- [Grafana 7.3.2 release notes]({{< relref "../release-notes/release-notes-7-3-2.md" >}})
+- [Grafana 7.3.3 release notes]({{< relref "../release-notes/release-notes-7-3-3.md" >}})
+- [Grafana 7.3.4 release notes]({{< relref "../release-notes/release-notes-7-3-4.md" >}})
+- [Grafana 7.3.5 release notes]({{< relref "../release-notes/release-notes-7-3-5.md" >}})
+- [Grafana 7.3.6 release notes]({{< relref "../release-notes/release-notes-7-3-6.md" >}})
+- [Grafana 7.3.7 release notes]({{< relref "../release-notes/release-notes-7-3-7.md" >}})
```

Comment:
Can we link to the root release notes page instead, patch releases & release notes are heavily automated and this would get quickly outdated (ie missing new patch releases)

Reply:
That is a good point, but OTOH, this saves users a click and scanning the page with the whole list of patch releases to find the one they need. The writers will be responsible for maintaining this, and odds are decent that the community will help out as well if we are slow. Let's try it this way for a while. We can change if it turns out to be unfeasible.

Reply:
@achatterjee-grafana, what are your thoughts?

Reply:
@oddlittlebird, less clicking is a always better user experience. But if the links are out of date quickly, it can become a maintenance issue. Let us see how this goes, as you said we can make adjustments as needed.